Testing code for reset.R and unit-test for reset functions

kar-agg-gen opened the pull request.

kar-agg-gen committed:
Updating the resetDGEobj function

kar-agg-gen committed:
Adding test for resetDGEobj

kar-agg-gen committed:
Removing the platformType parameter and updating tests

kar-agg-gen committed:
increasing the code coverage

saranya-ag commented on tests/testthat/test_reset.R:
```diff
@@ -2,5 +2,40 @@ context("DGEobj - tests for reset.R functions")
+                 regexp = "Required attribute \"PlatformType\" is missing.",
+                 fixed  = TRUE)
+
+    # checking DGEobj with wrong counts matirx
```
Rephrase comment to Testing DGEobj without count Matrix. 
Typo - Matrix

saranya-ag commented on tests/testthat/test_reset.R:
```diff
@@ -2,5 +2,40 @@ context("DGEobj - tests for reset.R functions")
+                 fixed  = TRUE)
+
+    # checking DGEobj with wrong data
+    names(test_DGEobj) <- c( "counts_orig" , "counts", "design_orig" , "design", "peptideAnnotation_orig")
```
Extra space before counts_orig

saranya-ag commented on tests/testthat/test_reset.R:
```diff
@@ -2,5 +2,40 @@ context("DGEobj - tests for reset.R functions")
+                 fixed  = TRUE)
+
+    # checking valid object
+    names(test_DGEobj) <- c( "counts_orig" , "counts", "design_orig" , "design", "isoformData_orig")
```
Extra space before counts_orig

saranya-ag commented on tests/testthat/test_reset.R:
```diff
@@ -2,5 +2,40 @@ context("DGEobj - tests for reset.R functions")
+
+    # checking valid object
+    names(test_DGEobj) <- c( "counts_orig" , "counts", "design_orig" , "design", "isoformData_orig")
+    # Expecting warning as current object could not build the GRanges object!
```
Rephrase comment 

Warning expected as the current object cannot build the GRanges object.

saranya-ag commented on R/reset.R:
```diff
@@ -26,24 +24,21 @@
-        counts <- getItem(dgeObj, "counts_orig")
-
-    design <- getItem(dgeObj, "design_orig")
+    metaList <- getBaseType(dgeObj, "meta")[1:3]
```
Can you check this - MetaList is not being used anywhere

kar-agg-gen committed:
Updating comments and removing unused variable

kar-agg-gen committed:
Updating tests

Merge branch 'develop' into increase-covr

# Conflicts:
#	tests/testthat/test_reset.R

kar-agg-gen committed:
Merge pull request #11 from kar-agg-gen/increase-covr

increasing the code coverage

klmedeiros-ag committed:
Develop (#8)

* added better assert_that messages addItem

* added useful assert_that messages as.R

* added assert_that messages attributes.R

* added meaningful assertthat messages getItem.R

* get rid of asserthat import as it isn't used here index.R

* added assertthat messages initDGEobj

* added assertthat messages mergeDGEobj

* added assertthat messages resetDGEobj

* added assertthat messages testToDGEobj

* added assertthat messages utils.R

* added explicit function calls assertthat::

* fixed showTypes to display df if pretty = FALSE

* fixed test for showTypes when pretty = FALSe

* get rid of debug doc addItem

* added setAttribute singular function

* added test for setAttribute singular

* export setAttribute

* update manuals

* generalize annotateDGEobj funct

* move subset functs

* update my test for setAttribute to use getAttribute singular

* update manuals

* adding missing assertthat message additem

* added missing assertthat message attributes

* added explicit call to assertthat annotate

* added assertthat message subset

* added assertthat messages rmItem

* added assertthat messages types

* move functions that are now in other places

* implement my showTYpes fix

* get rid of fucnts that are in new place now

* get rid of duplicate function

* update manuals

* consistency baseType(s)

* consistency baseType

* removed reference to Omicsoft

* remove explicit DGEobj::

* fixed spacing in tests

* get rid of "Be sure" in the assertthat messages

* update annotateDGEobj to accept a named list also

* added tests for passing named list to annotateDGEobj

* update some of the docs to annotateDGEobj

* part of the work fixing setAttribute(s)

* work to fix setAttribute/s

* update imports

* fix setAttributes docs

* update manuals

* addressing comments on https://github.com/cb4ds/DGEobj/pull/8

* update manuals

* update messages in test_init.R based on travis fails

cb4ds committed:
Merge branch 'develop' into develop

kar-agg-gen committed:
merge back from develop branch

Merge remote-tracking branch 'DGEobj-upstream/develop' into develop

# Conflicts:
#	R/reset.R

kar-agg-gen committed:
Updating function and messages

kar-agg-gen committed:
Merge branch 'develop' into fix-merge-kar

kar-agg-gen committed:
Merge pull request #12 from kar-agg-gen/fix-merge-kar

fixing merge issue

kar-agg-gen committed:
Increasing code coverage and updating resetDGEobj function. (#10)

* Updating the resetDGEobj function

* Adding test for resetDGEobj

* Removing the platformType parameter and updating tests

* increasing the code coverage

* Updating comments and removing unused variable

* Updating function and messages

Co-authored-by: Dr. Connie Brett <cb4ds@users.noreply.github.com>

kar-agg-gen committed:
Merge remote-tracking branch 'DGEobj-upstream/develop' into develop

kar-agg-gen committed:
Updating function and unit-test

kar-agg-gen committed:
Updating functions and unit-tests doe reset and addItem function (#11)

* Updating the resetDGEobj function

* Adding test for resetDGEobj

* Removing the platformType parameter and updating tests

* increasing the code coverage

* Updating comments and removing unused variable

* Updating function and messages

* Updating function and unit-test

Co-authored-by: Dr. Connie Brett <cb4ds@users.noreply.github.com>

cb4ds committed:
subset evaluation now handles symbols (#12)

cb4ds committed:
Remove merge and convert functions, update dependencies (#13)

* wip - removed plus dependencies

* updates after removal

cb4ds committed:
Merge branch 'cran' into develop

cb4ds committed:
Object creation (#14)

* wip - removed plus dependencies

* updates after removal

* removed as dependency for clarity of workflow

* updates to use the NEW, properly created test object.

* refactoring variable names used in testing for less confusion

* refactored to a small subsetted DGEobj for testing and examples

kar-agg-gen committed:
Merging back from upstream

Merge remote-tracking branch 'DGEobj-upstream/develop' into develop

# Conflicts:
#	tests/testthat/test_addItem.R
#	tests/testthat/test_reset.R

kar-agg-gen commented:
closing PR as changes are already merged
